WebApiGateway/WebApiGateway.Api/Clients/DecisionClient.cs

```diff
@@ -58,9 +65,12 @@ private async Task ConfigureHttpClientAsync()
     {
         var userId = _httpContextAccessor.HttpContext.User.UserId();
         var userAccount = await _accountServiceClient.GetUserAccount(userId);
```

Comment:
I think this is change in behaviour, previously you would get exception if organisation was empty or null
now it will silently ignore it which may not be intented behaviour, unless we can prove it wont cause any issue down stream we should look into this

Reply:
@hardik-desai-kcl this was changed as part of this line var organisation = userAccount.User.Organisations.First(); where its throwing SQ issue we should use index not First(). I changed to Index [0]. Now if no organisation found then it will throw exception at   _httpClient.DefaultRequestHeaders.AddIfNotExists("OrganisationId", organisation.Id.ToString());  which will break the code. That's the reason I have add this null check.

Old code also breaks if no organisation found.

I think we need to handle null check here as we are here.